### ISSUE TITLE
NN execution in script

### ISSUE THREAD

**Issue by asalvadorsalas** [asalvadorsalas/IFAE_NNTools#9]
Need to develop the NN as a script which saves in a very nice way the AUC/Separation so its easy to parse

Inputs should be:
pandas input
directories
name of the job
hyperparameters: nodes, layers, learning rate, dropout, list of variables???
